package.json

```diff
@@ -13,14 +13,14 @@
   "author": "Lucas Braun",
   "license": "MIT",
   "dependencies": {
-    "angular": "^1.6.8",
+    "angular": "^1.6.9",
```

Comment:
Why are you changing the angular version?

Reply:
I issued npm install angular by mistake instead of just npm install.  use the original one